spec/controllers/v2/tailorings_controller_spec.rb

```diff
@@ -86,38 +86,36 @@
         let(:item) do
           FactoryBot.create(
             :v2_tailoring,
-            rules: [], # no tailored rules
+            rules: [], # no tailored, but deselected rules
             value_overrides: [], # no tailored values
             policy: parent,
             os_minor_version: 8
           )
         end
 
         it 'returns XCCDF tailoring file with unselected rules' do
-          get :tailoring_file, params: extra_params.merge(parents: [:policy], format: :xml)
-
-          tailoring_file = Nokogiri::XML(response.body).remove_namespaces!
-          deselected, selected = tailoring_file.xpath('//Profile/select')
-                                               .partition { |sel| sel.attributes['selected'].value == 'false' }
+          get :tailoring_file, params: extra_params.merge(parents: [:policy], format: format)
 
           expect(response).to have_http_status :ok
-          expect(deselected.map { |r| r.attributes['idref'].value }).to match_array(item.profile.rules.map(&:ref_id))
-          expect(selected).to match_array(item.rules.map(&:ref_id))
+          expect(values).to be_empty
+          expect(selected_rules).to be_empty
+          expect(deselected_rules).not_to be_empty
+          expect(deselected_rule_ref_ids).to match_array(item.profile.rules.map(&:ref_id))
         end
       end
 
       context 'with default rules and values' do
         let(:item) do
           FactoryBot.create(
             :v2_tailoring,
-            :with_default_rules, # no tailored rules
+            :with_default_rules, # no tailored, no deselected rules
```

Comment:
IMO these comments should be part of the context descriptions, if they aren't already.